netneurotools/networks.py

```diff
@@ -71,8 +79,13 @@ def func_consensus(data, n_boot=1000, ci=95, seed=None):
 
     # generate `n_boot` bootstrap correlation matrices by sampling `t` time
     # points from the concatenated time series
+    if isinstance(data, list):
+        nsample = int(collapsed_data.shape[-1] / len(data))
+    else:
+        nsample = data.shape[1]
```

Comment:
Sorry to ask more of you, but do you think we could move this into the same if-else clause as above where we generate `collapsed_data` (lines 69-72)? So, right after you generated the `collapsed_data` array you will also generate `nsample`! No need for the duplicate if-else check.